src/validation.h

```diff
@@ -430,13 +430,26 @@ class CVerifyDB {
 /** Replay blocks that aren't fully applied to the database. */
 bool ReplayBlocks(const CChainParams& params, CCoinsView* view);
 
+// GetBlockIndex(...) should be used instead if the caller assumes that the
```

Comment:
Maybe improve a bit:
```cpp
 /** Find and retrieve the block index for the given block hash.
  *  The caller must check if the block index is valid. If the block index
  *  always exists then use GetBlockIndex instead.
  *  Call with cs_main held.
  *
  *  @param[in] hash The block hash
  *  @return The block index pointer if found, nullptr otherwise
  *  @see GetBlockIndex
  */